_rules/SC2-5-3-name-from-content-included-in-accessible-name.md

```diff
@@ -0,0 +1,176 @@
+---
+name: Name from content included in accessible name
+description: |
+  Interactive elements labelled through their content must have their visible label as part of their accessible name
+
+success_criterion:
+- 2.5.3 # Label in Name
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+- Bryn Anderson
+- Jey Nandakumar
+---
+
+## Test procedure
+
+### Applicability
+
+This rule applies to any HTML or SVG element that:
+* has a [semantic role](#semantic-role) that is a [widget](https://www.w3.org/TR/wai-aria-1.1/#widget_roles) that [supports name from content](https://www.w3.org/TR/wai-aria-1.1/#namefromcontent), and 
+* has [visible](#visible) [text](https://www.w3.org/TR/WCAG21/#dfn-text), and
+* where the [visible](#visible) [text](https://www.w3.org/TR/WCAG21/#dfn-text) does not [match](#matching-characters) the [accessible name](#accessible-name).
+
+**Note**: [widget roles](https://www.w3.org/TR/wai-aria-1.1/#widget_roles) that [support name from content](https://www.w3.org/TR/wai-aria-1.1/#namefromcontent) are: `button`, `checkbox`, `gridcell`, `link`, `menuitem`, `menuitemcheckbox`, `menuitemradio`, `option`, `radio`, `searchbox`, `switch`, `tab`, `treeitem`.
+
+### Expectation
+
+The complete [visible](#visible) [text](https://www.w3.org/TR/WCAG21/#dfn-text) of the target element is [included](#included-characters) within its [accessible name](#accessible-name).
+
+## Assumptions
+
+This rule assumes that the [visible](#visible) [text](ttps://www.w3.org/TR/WCAG21/#dfn-text) has the same order as the text in the code, e.g. CSS has not been used to re-arrange pieces of text so that they are presented in a different order.
```

Comment:
This may not be needed anymore.

Reply:
@kasperisager, You sure? I added it along with another bunch of comments from our discussions. Happy to drop it, though.

Reply:
My bad, the rule of course talks about text as it appears in code now, which we must assume is about the same as it appears on screen 🙈